src/components/Avatar/styled/avatar.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components/native';
+import { BaseProps } from '../../types';
+import { COLOR_BRAND, COLOR_WHITE } from '../../../styles/colors';
+
+interface Props extends BaseProps {
+    size?: string;
+}
+
+const StyledAvatar = styled.View<Props>`
+    position: relative;
```

Comment:
I think that we should avoid the use of `position: absolute` or `position: relative`. Since we are using `flex` it should not be necessary to use `position`.
If this is necessary for a specific use case we can add it in the future